Jamulus.pro

```diff
@@ -1168,3 +1168,12 @@ contains(CONFIG, "disable_version_check") {
 }
 
 ANDROID_ABIS = armeabi-v7a arm64-v8a x86 x86_64
+
+# Enable formatting all code via `make clang_format`.
+# Note: When extending the list of file extensions or when adding new code directories,
+# be sure to update .github/workflows/coding-style-check.yml and .clang-format-ignore as well.
+CLANG_FORMAT_SOURCES = $$files(*.cpp, true) $$files(*.h, true)
+CLANG_FORMAT_SOURCES = $$find(CLANG_FORMAT_SOURCES, ^\(android|ios|mac|linux|src|windows\)/)
+CLANG_FORMAT_SOURCES ~= s!^\(windows/nsProcess/|src/res/qrc_resources\.cpp\)\S*$!!g
```

Comment:
People on Windows will likely have `ASIOSDK2` under `windows` when building locally -- if they use `make clang-format`, that wants ignoring.